change/@fluentui-react-nav-preview-35207415-1026-46cd-8f89-9fb2f7527ee7.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "'feat: Initial preview publication",
```

seem you haven't use the generator.

Please use automation we have in place for initial releases based on phase.

`yarn nx g @fluentui/workspace-plugin:prepare-initial-release`

```suggestion
  "comment": "feat: release preview package",
```

---

I used the generator you pointed me to in the component implementation guide.

---

What specifically is technically different between the result of that script vs what is in place now?

---

well the automation uses different wording, that's why I mentioned that you probaby didn't use it as I mentioned before

source:

https://github.com/microsoft/fluentui/blob/master/tools/workspace-plugin/src/generators/prepare-initial-release/index.ts#L87

test:
the outputs are obviously tested so we can guarantee things work https://github.com/microsoft/fluentui/blob/master/tools/workspace-plugin/src/generators/prepare-initial-release/index.spec.ts#L219-L221

---

Ah - I did adjust the wording of the change file as it didn't quite read correctly to me.